### PR TITLE
Prefer common field names.

### PR DESCRIPTION
**Pull request by dehora**
This provides a description of four common properties (id,
created, modified, type) in terms of what they mean and
what their syntax and names are. Locally, people may prefer
different names and values, globally it's a bikeshed that
offers no benefit to end users of APIs.

For #79

### CODE REVIEW

File: json-guidelines/JsonGuidelines.md [hunk 1]
```diff
@@ -12,6 +12,20 @@ No established industry standard exists, but many popular Internet companies pre
 
 Property names are restricted to ASCII encoded strings. The first character must be a letter, an underscore  or a dollar sign, and subsequent characters can be a letter, an underscore, a dollar sign, or a number.
 
+### {{ book.must }} Prefer common field names
```
Comment (ePaul) [inline json-guidelines/JsonGuidelines.md]: "Must" + "prefer" sounds contradictory. But I don't know how to word this better.

Comment (tfrauenstein) [inline json-guidelines/JsonGuidelines.md]: 👍  
+editorial change proposal: let's move it to "14. Common Data Objects *and Fields*" Section

Comment (dehora) [inline json-guidelines/JsonGuidelines.md]: > let's move it to 14

done in b998fe3

Comment (whiskeysierra) [inline json-guidelines/JsonGuidelines.md]: > But I don't know how to word this better.

What about _**Use** common field names_?

File: json-guidelines/JsonGuidelines.md [hunk 2]
```diff
@@ -12,6 +12,20 @@ No established industry standard exists, but many popular Internet companies pre
...
+
+There are some data fields that come up again and again in API data. We describe four here   
+
+- `id`: the identity of the object. If used, IDs must opaque strings and not numbers. IDs are unique within some documented context, are stable and don't change for a given object once assigned, and are never recycled cross entities. 
+
+- `created`: when the object was created. If used this must be a date construct.
```
Comment (whiskeysierra) [inline json-guidelines/JsonGuidelines.md]: Maybe *date-time*?

**Comment by whiskeysierra** [comment]
:+1:

**Comment by dehora** [comment]
had to reflog and rebase again to get 6c32ca8

**Comment by ePaul** [comment]
:+1:

**Comment by fmueller** [comment]
👍

**Pull request merged**